src/styles/colors.scss

```diff
@@ -8,21 +8,6 @@ $category: #04669a;
 $tag: #048b9a;
 $tracker: #c97d09;
 
-// State filter colors
```

Comment:
why get rid of the colors?

Reply:
It was only used for the active filters state chip, which isn't needed anymore because we now filter by torrent state and values differs between these 2